config/webpack/webpack.config.js

```diff
@@ -70,6 +71,13 @@ module.exports = function (dirname, config = {}) {
             new CircularDependencyPlugin({
                 exclude: /(node_modules|examples)\/./,
                 failOnError: false // https://github.com/nodejs/readable-stream/issues/280#issuecomment-297076462
+            }),
+            new GlobalizePlugin({
+                production: false,
+                developmentLocale: "en",
+                supportedLocales: ["en", "es", "fr"],
+                messages: "../../src/i18n/[locale].json",
```

Comment:
There is another solution:
Instead of having the relative path define for the messages, we can remove that "optional" definition from webpack.config.js file, and load the message from each file using the Globalization. 
Pros: 
    - no path (relative or absolute path definition)
    - all string definition would be loaded from the file using it 
    - User can see all languages definition used in this specific file, all variables that could be translated.
    - No need to create multi-levels for the variable definition since the variables are only used in this file.
Cons:
    - Not using a file having all strings definition at the same place for a package.
    - When adding a new language, the user will need to go through all files having Globalization and add a definition in each separate file for each variable. (Might be harder to maintain) instead of creating a new file for a package and adjust all strings in that new file.

Ex:, in a file, just put the globalization.loadMessages (all supported languages) at the beginning of the file:
Globalize.loadMessages({
    "en": {
        "navigator": {
            "browser": {
                "menu": {
                    "OpenWith": "Open With"
                }
            }
        }
    },
    "es": {
        "navigator": {
            "browser": {
                "menu": {
                    "OpenWith": "Abrir con"
                }
            }
        }
    },
    "fr": {
        "navigator": {
            "browser": {
                "menu": {
                    "OpenWith": "Ouvrir avec"
                }
            }
        }
    }
});

If we use this approach by having the string definition for this file, then we should remove the multi-levels string definition and it becomes:
Globalize.loadMessages({
    "en": {
        "OpenWith": "Open With"
        }
    },
    "es": {
        "OpenWith": "Abrir con"
        }
    },
    "fr": {
        "OpenWith": "Ouvrir avec"
        }
    }
});

What should be the preferred solution ?